modules/gcloud/src/test/java/org/testcontainers/containers/DatastoreEmulatorContainerTest.java

```diff
@@ -40,6 +40,20 @@ public void testSimple() {
 
         assertThat(datastore.get(key).getString("description")).isEqualTo("my description");
     }
+
     // }
 
+    @Test
+    public void testWithFlags() {
+        try (
+            DatastoreEmulatorContainer emulator = new DatastoreEmulatorContainer(
+                "gcr.io/google.com/cloudsdktool/cloud-sdk:367.0.0-emulators"
+            )
+                .withFlags("--consistency 1.0")
+        ) {
+            emulator.start();
+
+            assertThat(emulator.getContainerInfo().getConfig().getCmd()).anyMatch(e -> e.contains("--consistency 1.0"));
```

Comment:
In an ideal situation, we would not test for changing the CMD, but instead, test that the application was started with the flag. But not sure if worth the effort and feasible.

Reply:
couldn't find any other way to test it 😅